docs/resources/domain.md

```diff
@@ -18,6 +18,7 @@ resource "mailgun_domain" "default" {
   region        = "us"
   spam_action   = "disabled"
   smtp_password   = "supersecretpassword1234"
+  dkim_key_size   = 2048
```

Comment:
Just to not cause confusion in the docs, probably best that this value be assigned the default `1024` value.

Reply:
yeah, true! Changing it.